src/AuthProvider.tsx

```diff
@@ -23,25 +24,31 @@ export const AuthProvider: React.FC<PropsWithChildren> = ({children}) => {
       login: async (email: string) => {
         try {
           console.log(`Trying to login with email: ${email}`);
+          sampleRUM('chat:login', { source: 'attempt' });
```

Comment:
Our `source` values look like CSS selectors, so either
```suggestion
          sampleRUM('chat:login', { source: '.attempt' });
```

or 
```suggestion
          sampleRUM('chat:login', { source: '#attempt' });
```

would work better